### DISCUSSION TITLE
OFX exports:  "FITID" (unique id) is changing between exports on different dates

### DISCUSSION THREAD

**Discussion by adejongh** [Investec-Developer-Community/.github#11] [General]
I've come across this a few times now - since the Investec to Xero integration ~is currently~ was broken for my account (~investigation ongoing for more than 2 months now...~ see https://github.com/orgs/Investec-Developer-Community/discussions/13) I have to export OFX files, and import them into Xero.  This means I have to "overlap" the exports by one day to make sure all transactions are imported into Xero.

So, for example, export ending on 15 March 2024, there is this transaction:
```
<STMTTRN>
          <TRNTYPE>DEBIT
          <DTPOSTED>20240315120000
          <DTUSER>20240315120000
          <TRNAMT>-180.00
          <FITID>0067774860
          <NAME>FE97PR4ECMJQSRP3JLRA
</STMTTRN>
``` 
Running the export tonight, starting on 15 March 2024:
```
<STMTTRN>
          <TRNTYPE>DEBIT
          <DTPOSTED>20240315120000
          <DTUSER>20240315120000
          <TRNAMT>-180.00
          <FITID>0068254435
          <NAME>FE97PR4ECMJQSRP3JLRA
</STMTTRN>
```

Why does the "FITID" change?  It should stay unique forever?

Search for "FITID" in https://financialdataexchange.org/common/Uploaded%20files/OFX%20files/OFX%20Banking%20Specification%20v2.3.pdf

**Comment by christogreeff** [comment]
> An FI (or its Service Provider) assigns an <FITID> to uniquely identify a financial transaction that can 
appear in an account statement. Its primary purpose is to allow a client to detect duplicate responses.

So, yeah. If it changes you're going to have issues. Seems to be a problem with the export.

**Comment by programmable-banking-community** [comment]
Hi @adejongh, thanks for this. I'm assuming this is for a Private account? We'll take a look at this and get back to soon as possible. Thanks  again :)

**Comment by adejongh** [comment]
This happens with a private business account.  Happy to share the specific OFX exports directly if that will help.
